recipe/build.sh

```diff
@@ -10,13 +10,14 @@ if [ "${target_platform}" == "linux-ppc64le" ] || [ "${target_platform}" == "lin
 fi
 
 # Numpy cannot be found in ppc64le for some reason... some extra help will do ;)
+# "core" for np v1; "_core" for v2.
 EXTRA_CMAKE_FLAGS=""
-if [ "${target_platform}" == "linux-ppc64le" ]; then
-    EXTRA_CMAKE_FLAGS+=" -D PYTHON_NUMPY_INCLUDE_PATH=${SP_DIR}/numpy/core/include"
+if [[ "${target_platform}" == "linux-ppc64le" || "${target_platform}" == "linux-aarch64" ]]; then
+    EXTRA_CMAKE_FLAGS+=" -D Python3_NumPy_INCLUDE_DIR=${SP_DIR}/numpy/_core/include"
```

Comment:
```suggestion
    EXTRA_CMAKE_FLAGS+=" -D Python3_NumPy_INCLUDE_DIR=$(python -c 'import numpy as np; print(np.get_include())'"
```

this is more correct, always.

Reply:
More info https://github.com/conda-forge/conda-forge.github.io/issues/2260

Reply:
you could if you wanted to define this numpy_include_dir always as well.